07-hosting.Rmd

```diff
@@ -341,7 +341,8 @@ Unfortunately, pm2 doesn't understand R scripts natively; however, it is possibl
 
 library(plumber)
 pr <- plumb('myfile.R')
-pr$run(port=4000)
+pr$run(port=4000, host="0.0.0.0") 
+#Setting the host option on a VM instance ensures the application can be accessed externally. This may be only true for Linux users.
```

Comment:
```suggestion
# Setting the host option on a VM instance ensures the application can be accessed externally.
# (This may be only true for Linux users.)
```